test-suite/Makefile

```diff
@@ -57,8 +57,16 @@ BIN:=$(COQPREFIX)/bin/
 # An alternative is ifeq ($(OS),Windows_NT) using make's own variable.
 ifeq ($(ARCH),win32)
   export FINDLIB_SEP=;
+
+  # in output tests we call "rocq c output/foo.v"
+  # this gets turned into (Filename.concat current_dir_name "output/foo.v")
+  # (Coqcargs.add_compile)
+  # for test stability across platforms we turn the \ from Filename.concat into a /
```

Comment:
an alternative would be to call `rocq c ./output/foo.v`, then the leading `./` should be kept intact